bolt-micronaut/pom.xml

```diff
@@ -11,7 +11,7 @@
 
     <properties>
         <micronaut.bom.version>2.0.2</micronaut.bom.version>
```

Comment:
I intentionally avoided upgrading this - https://github.com/micronaut-projects/micronaut-core/issues/3933#issuecomment-698046178